relay-server/src/metrics_extraction/transactions/mod.rs

```diff
@@ -634,7 +634,7 @@ mod tests {
                     "bd429c44b67a3eb4",
                 ),
                 trace_id: TraceId(
-                    "ff62a8b040f340bda5d830223def1d81",
+                    ff62a8b0-40f3-40bd-a5d8-30223def1d81,
```

Comment:
Why are `-`s being introduced in these snapshots?

Reply:
Yep, I will do this! Good idea, so that it keeps being consistent, should we also use the `""`? Might be unnecessary but idk

Reply:
Oh, duh, I missed that this is `Debug`, not serialization. Still, I would also agree with overriding the `Debug` impl.

Reply:
`TraceId(abcd...)` should be fine, don't really see a need for the `"` but nbd either way